1209/8787/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: MVMC-lab_M128_V3
+owner: MVMC-lab
+license: NCU-MVMC-lab
+site: https://gitlab.com/MVMC-lab
+source: https://gitlab.com/MVMC-lab
```

Comment:
Please link to the specific repository for the hardware. It should link to the software.